### PR TITLE
New Chromebook Cheat Sheet

### PR DESCRIPTION
**Pull request by numrut**
IA Page: https://duck.co/ia/view/chromebook_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@numrut Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Chromebook Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/chromebook_cheat_sheet_mobile.png?Ofs9ie5NkT)

File: share/goodie/cheat_sheets/json/chromebook.json [hunk 1]
```diff
@@ -0,0 +1,222 @@
+{
+    "id": "chromebook_cheat_sheet",
+    "name": "Chromebook Cheat Sheet",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/chromebook.json]: Lets change name to "Chromebook"

File: share/goodie/cheat_sheets/json/chromebook.json [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+    },
+
+    "aliases": [
+        "chrome os"
+    ],
+
+    "template_type": "keyboard",
+
+    "section_order": [
+        "Tabs and windows",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/chromebook.json]: Tabs and Windows

File: share/goodie/cheat_sheets/json/chromebook.json [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+        "Page shortcuts",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/chromebook.json]: Page Shortcuts

File: share/goodie/cheat_sheets/json/chromebook.json [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+        "Browser settings",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/chromebook.json]: Browser Settings

File: share/goodie/cheat_sheets/json/chromebook.json [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+        "Text editing"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/chromebook.json]: Text Editing

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Chromebook Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/chromebook_cheat_sheet_mobile.png?gTajJe9TQZ)

**Comment by pjhampton** [comment]
Closing as this PR looks to have gone stale.

If you are looking for something more substantial to work through that will have a big impact for the users of DuckDuckGo, please check out the [DuckDuckHack Issues Table](https://duckduckhack.com/issues). 🚀  😄

**Pull request closed without merging**